### ISSUE TITLE
Android 14 support

### ISSUE THREAD

**Issue by KristapsKrumins** [SumSubstance/idensic-mobile-sdk-sample-android#14]
```
Fatal Exception: java.lang.RuntimeException: Unable to resume activity {/com.sumsub.sns.presentation.screen.SNSAppActivity}: java.lang.SecurityException: One of RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED should be specified when a receiver isn't being registered exclusively for system broadcasts
       at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4962)
       at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4995)
       at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:57)
       at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
       at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:180)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:98)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loopOnce(Looper.java:205)
       at android.os.Looper.loop(Looper.java:294)
       at android.app.ActivityThread.main(ActivityThread.java:8177)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
```

**Comment by palmut** [comment]
Thank you for submitting the issue. It will be fixed in next releases.

**Comment by kk-atom** [comment]
Its November, still not fixed.

**Comment by palmut** [comment]
The issue is fixed in Sumsub Android SDK 1.28.0
